components/LoginForm.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from '../styles/Login.module.css'
+
+interface LoginForm {
+  onSubmit: React.FormEvent,
+  buttonText: string,
+}
+
+function LoginForm({onSubmit, buttonText}) {
+  function handleSubmit(event) {
+    event.preventDefault()

```

Comment:
Why do you need to preventDefault() here?

Reply:
This is supposed to be on a modal based on epic react but I decided to do my own version of the form. I may have forgotten to remove it. Sorry about that sir.